fala/templates/adviser/results.html

```diff
@@ -95,25 +95,25 @@ <h1 class="govuk-heading-xl govuk-!-margin-bottom-5">{{_('No search results')}}<
     {% endif %}
 
     <span class="govuk-body results-header">
-    {% if data.count > 0 %}
-      {% if data.origin %}
-        {% trans page_count=data.results|length, result_count=data.count %}
-          {% pluralize %}
-          <span class="govuk-!-font-weight-bold" id="result-count-overall">{{ result_count }} results</span> in order of closeness to
-        {% endtrans %}
-        <strong class="notranslate" translate="no">{{ form.postcode.value()|upper }}</strong>
-        {%- if form.name.value() %}
-          {{ _('matching') }} <strong>{{ form.name.value() }}</strong>.
-        {%- else -%}
-          .
+      {% if data.count > 0 %}
```

Comment:
was this just a linting change?

Reply:
Not a linting change. I had to refactor template, so that the test was pulled out when you run the `makemessages` command.